<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerVolumeRenderingModule</class>
 <widget class="qSlicerWidget" name="qSlicerVolumeRenderingModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Volume Render</string>
  </property>
  <layout class="QFormLayout" name="formLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="VolumeNodeLabel">
     <property name="text">
      <string>Volume:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="qMRMLNodeComboBox" name="VolumeNodeComboBox">
     <property name="nodeTypes">
      <stringlist>
       <string>vtkMRMLScalarVolumeNode</string>
      </stringlist>
     </property>
     <property name="addEnabled">
      <bool>false</bool>
     </property>
     <property name="renameEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="ctkCollapsibleButton" name="InputsCollapsibleButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Inputs</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="DisplayNodeLlabel">
        <property name="text">
         <string>Display:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="qMRMLNodeComboBox" name="DisplayNodeComboBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLVolumeRenderingDisplayNode</string>
         </stringlist>
        </property>
        <property name="renameEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="ROINodeLabel">
        <property name="text">
         <string>ROI:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="qMRMLNodeComboBox" name="ROINodeComboBox">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLAnnotationROINode</string>
         </stringlist>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
        <property name="renameEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="VolumePropertyNodeLabel">
        <property name="text">
         <string>Property:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="qMRMLNodeComboBox" name="VolumePropertyNodeComboBox">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLVolumePropertyNode</string>
         </stringlist>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
        <property name="renameEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="ViewNodeLabel">
        <property name="text">
         <string>View:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="VisibilityLabel">
        <property name="text">
         <string>Visible:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="VisibilityCheckBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="qMRMLCheckableNodeComboBox" name="ViewCheckableNodeComboBox">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLViewNode</string>
         </stringlist>
        </property>
        <property name="baseName">
         <string>View</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="ctkCollapsibleButton" name="RenderingCollapsibleButton">
     <property name="text">
      <string>Rendering</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="contentsFrameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="RenderingTabWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="TechniquesTab">
         <attribute name="title">
          <string>Techniques</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="VolumePropertyTab">
         <attribute name="title">
          <string>Volume Properties</string>
         </attribute>
         <layout class="QVBoxLayout" name="volumePropertyLayout">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="qMRMLVolumePropertyNodeWidget" name="VolumePropertyNodeWidget"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="ROITab">
         <attribute name="title">
          <string>ROI</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="qMRMLAnnotationROIWidget" name="ROIWidget"/>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="MiscTab">
         <attribute name="title">
          <string>Misc</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLCheckableNodeComboBox</class>
   <extends>qMRMLNodeComboBox</extends>
   <header>qMRMLCheckableNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLVolumePropertyNodeWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLVolumePropertyNodeWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLAnnotationROIWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLAnnotationROIWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../Annotation/Resources/qSlicerAnnotationModule.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>VolumePropertyNodeComboBox</sender>
   <signal>currentNodeChanged(vtkMRMLNode*)</signal>
   <receiver>VolumePropertyNodeWidget</receiver>
   <slot>setMRMLVolumePropertyNode(vtkMRMLNode*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>519</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ROINodeComboBox</sender>
   <signal>currentNodeChanged(vtkMRMLNode*)</signal>
   <receiver>ROIWidget</receiver>
   <slot>setMRMLAnnotationROINode(vtkMRMLNode*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>493</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>358</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>VolumeNodeComboBox</sender>
   <signal>currentNodeChanged(bool)</signal>
   <receiver>InputsCollapsibleButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DisplayNodeComboBox</sender>
   <signal>currentNodeChanged(bool)</signal>
   <receiver>ROINodeComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DisplayNodeComboBox</sender>
   <signal>currentNodeChanged(bool)</signal>
   <receiver>VolumePropertyNodeComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DisplayNodeComboBox</sender>
   <signal>currentNodeChanged(bool)</signal>
   <receiver>VisibilityCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>412</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DisplayNodeComboBox</sender>
   <signal>currentNodeChanged(bool)</signal>
   <receiver>ViewCheckableNodeComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>VolumeNodeComboBox</sender>
   <signal>currentNodeChanged(bool)</signal>
   <receiver>InputsCollapsibleButton</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerVolumeRenderingModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ViewCheckableNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerVolumeRenderingModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ROINodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>438</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>512</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerVolumeRenderingModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>VolumePropertyNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>548</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>606</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerVolumeRenderingModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>DisplayNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerVolumeRenderingModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>VolumeNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>501</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
